packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -115,20 +122,16 @@ interface IMajorityVoting {
     /// @return VoteOption of the requested voter for a certain vote.
     function getVoteOption(uint256 _voteId, address _voter) external view returns (VoteOption);
 
-    /// @param participationRequiredPct The required participation in percent.
-    /// @param supportRequiredPct The required support in percent.
-    /// @param minDuration The minimal duration of a vote.
-
     /// @notice Returns all information for a vote by its ID.
     /// @param _voteId The ID of the vote.
     /// @return open Wheter the vote is open or not.
     /// @return executed Wheter the vote is executed or not.
     /// @return startDate The start date of the vote.
     /// @return endDate The end date of the vote.
     /// @return snapshotBlock The block number of the snapshot taken for this vote.
-    /// @return supportRequired The support required.
-    /// @return participationRequired The required participation.
-    /// @return votingPower The voting power participating in the vote.
+    /// @return relativeSupportThresholdPct The relative support threshold in percent.
+    /// @return totalSupportThresholdPct The total support threshold in percent.
+    /// @return plenum The voting power participating in the vote.
```

Comment:
The wording relative, and plenum might be intimidating, do u think we can use some simpler wording ?

Reply:
I renamed plenum to census for now, which I think is also @brickpop's suggestion. 

`votingPower` is not correct/could be misleading (see its definition https://www.polyas.com/election-glossary/voting-power). 
Maybe `totalVotingPower` or, better, `eligibleVotingPower` would work.

I think the relative $\text{relative support} = N_\text{yes} / (N_\text{yes} + N_\text{no})$ and $\text{total support} = N_\text{yes} / N_\text{census}$ are already fitting quite well, and we should give the formulas as a tooltip on the UI
![Untitled1](https://user-images.githubusercontent.com/20623991/201302877-557a8e3b-5c8d-4369-b5bd-f793a521509d.png)
![Untitled2](https://user-images.githubusercontent.com/20623991/201303058-e478dfa9-5588-45a0-91eb-3559cac1b4a1.png)
![Untitled3](https://user-images.githubusercontent.com/20623991/201302971-277106ad-19ea-4d12-bcde-0e75ea9de764.png)

Reply:
We will do it here: https://aragonassociation.atlassian.net/browse/APP-1250